### PR TITLE
feat: #414 layout

### PR DESCRIPTION
**Pull request by wjtwzgl12**

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -1,265 +1,199 @@
...
-    }
-  }
-  
+  const toggleNavbarMenu = () => {
+    setBurgerActive(!burgerActive);
+  };
 
+  const switchLocale = () => {
+    const newLocale = locale === 'Engish' ? 'en' : 'zh';
+    setLocale(newLocale);
```
Comment (ucfyao) [inline src/app/layout.tsx:44]: Why is it local==='Engish '

File: src/app/layout.tsx [hunk 2]
```diff
@@ -1,265 +1,199 @@
...
+      router.push('/');
+    }
+  };
 
   return (
     <html lang="en">
-      <body className={inter.className}>
+      <body>
+        <div>
+          {/* Nav bar */}
```
Comment (ucfyao) [inline src/app/layout.tsx:63]: Please try encapsulating as different components.

File: src/assets/images/wechat-group.jpg
Comment (ucfyao) [inline src/assets/images/wechat-group.jpg:1]: This picture doesn't work, you can use mine.

**Review by ucfyao — COMMENTED** [review]
Further learn about components and layout.

File: src/app/layout.tsx [hunk 3]
```diff
@@ -1,265 +1,199 @@
...
-              </Link>
-            </div>
-          </div>
-
-          <div className="border-b test-white mx-10"></div>
-
-          <div className="text-center mx-10 ">
-            <p>©2023 AntClony All right reserved</p>
-          </div>
+          </footer>
```
Comment (Yingxue0323) [inline src/app/layout.tsx:194]: Could you extract the header(nav bar) and footer into components?

**Pull request closed without merging**